### DISCUSSION TITLE
Problem with Escaping Delimiter in Quotes

### DISCUSSION THREAD

**Discussion by flerach** [JoshClose/CsvHelper#2098] [Q&A] [answered]
Hello @JoshClose ,

I have a problem reading my data at this line:

`637460;„990000“;17.03.2021;„Payment“;„STRIPE22142“;„LKKWebshop Stripe #;xerox“;„40016“;345,00;„I3424“;„“;„LELEK\STU“;„Ja“;0,00;„Debitor“;180878;170,00;0,00;17.03.2021;„“;„BankAccount“;„G880“;„“;„“`

I usually use this config:

```
           Quote = '"',
           Delimiter = ";",
           Mode = CsvMode.RFC4180,

```

i already tried the escape mode but its not working.

![image](https://user-images.githubusercontent.com/36732590/205741259-5b340783-6358-4be9-807a-1ff604d49091.png)
I mean why it is breaking here?

`„LKKWebshop Stripe #;xerox“` 

It should be readed like => LKKWebshop Stripe #;xerox <= i dont get it i mean the semicolon is between the quotes?

**Comment by MiblaCz** [comment]
„LKKWebshop Stripe #;xerox“ is not the same as "LKKWebshop Stripe #;xerox"
You have to use the correct quote character.

**Comment by JoshClose** [comment, marked as answer]
The character `“` (8220 or left quotation mark) is not the same as `"` (34 quotation mark).
